### PR TITLE
Authorable Content: Tutorials Front Door

### PR DESCRIPTION
**Pull request by ashleemboyer**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-content-tutorials-front-door-hashicorp.vercel.app/tutorials) 🔎

### CODE REVIEW

File: src/content/tutorials-landing.json [hunk 1]
```diff
@@ -1,147 +1,163 @@
...
-				"description": "The Consul Associate Certification is for Site Reliability Engineers, Solutions Architects, DevOps professionals, or other Cloud Engineers who know the basic concepts and skills to build, secure, and maintain open source HashiCorp Consul."
-			},
 			"featuredUseCases": [
 				{
 					"href": "/consul/tutorials/kubernetes",
-					"text": "Service Mesh on Kubernetes"
+					"text": "Service mesh"
 				},
 				{
 					"href": "/consul/tutorials/microservices",
```
Comment (im2nguyen) [inline src/content/tutorials-landing.json:62]:
```suggestion
					"href": "/consul/tutorials/kubernetes",
```

File: src/content/tutorials-landing.json [hunk 2]
```diff
@@ -1,147 +1,163 @@
...
-			"certificationProgram": {
-				"slug": "networking-automation",
-				"title": "Verify your Networking Automation skills",
-				"description": "The Consul Associate Certification is for Site Reliability Engineers, Solutions Architects, DevOps professionals, or other Cloud Engineers who know the basic concepts and skills to build, secure, and maintain open source HashiCorp Consul."
-			},
 			"featuredUseCases": [
 				{
 					"href": "/consul/tutorials/kubernetes",
-					"text": "Service Mesh on Kubernetes"
+					"text": "Service mesh"
```
Comment (im2nguyen) [inline src/content/tutorials-landing.json:59]:
```suggestion
					"text": "Service discovery and health monitoring"
```

File: src/content/tutorials-landing.json [hunk 3]
```diff
@@ -1,147 +1,163 @@
...
 		},
 		"consul": {
-			"certificationProgram": {
-				"slug": "networking-automation",
-				"title": "Verify your Networking Automation skills",
-				"description": "The Consul Associate Certification is for Site Reliability Engineers, Solutions Architects, DevOps professionals, or other Cloud Engineers who know the basic concepts and skills to build, secure, and maintain open source HashiCorp Consul."
-			},
 			"featuredUseCases": [
 				{
 					"href": "/consul/tutorials/kubernetes",
```
Comment (im2nguyen) [inline src/content/tutorials-landing.json:58]:
```suggestion
					"href": "/consul/tutorials/developer-discovery",
```

**Review by im2nguyen — APPROVED** [review]
changed some of the ordering for the featured use cases, but other than that i like it 😄

**Review by im2nguyen — APPROVED** [review]
changed some of the ordering for the featured use cases, but other than that i like it 😄

**Review by tunzor — APPROVED** [review]
Nomad, Vagrant, and Waypoint look good 👍

File: src/content/tutorials-landing.json [hunk 4]
```diff
@@ -1,147 +1,163 @@
...
 				{
 					"href": "/boundary/tutorials/access-management",
-					"text": "Secure Access Management Workflows"
+					"text": "Secure access management workflows"
 				}
 			],
 			"featuredCollectionSlugs": [
 				"boundary/hcp-getting-started",
 				"boundary/oss-getting-started"
 			]
```
Comment (yhyakuna) [inline src/content/tutorials-landing.json:141]: Adding the 3rd collection:

```suggestion
			"featuredCollectionSlugs": [
				"boundary/hcp-getting-started",
				"boundary/oss-getting-started",
				"boundary/credential-management"
			]
```

File: src/content/tutorials-landing.json [hunk 5]
```diff
@@ -1,147 +1,163 @@
...
-				"vault/cloud"
+				"packer/docker-get-started",
+				"packer/aws-get-started",
+				"packer/hcp-get-started"
 			]
 		},
 		"boundary": {
 			"featuredUseCases": [
 				{
 					"href": "/boundary/tutorials/hcp-administration",
```
Comment (yhyakuna) [inline src/content/tutorials-landing.json:126]:
```suggestion
					"href": "/boundary/tutorials/credential-management",
```

Comment (yhyakuna) [inline src/content/tutorials-landing.json:126]:
```suggestion
					"href": "/boundary/tutorials/credential-management",
```

File: src/content/tutorials-landing.json [hunk 6]
```diff
@@ -1,147 +1,163 @@
...
-					"text": "HCP Boundary Resource Management Workflows"
+					"text": "Resource management"
 				},
 				{
 					"href": "/boundary/tutorials/oss-administration",
-					"text": "Self-Hosted Boundary Resource Management Workflows"
+					"text": "Self-hosted resource management workflows"
 				},
 				{
 					"href": "/boundary/tutorials/access-management",
```
Comment (yhyakuna) [inline src/content/tutorials-landing.json:134]:
```suggestion
					"href": "/boundary/tutorials/worker-management",
```

File: src/content/tutorials-landing.json [hunk 7]
```diff
@@ -1,147 +1,163 @@
...
-					"text": "Secure Access Management Workflows"
+					"text": "Secure access management workflows"
```
Comment (yhyakuna) [inline src/content/tutorials-landing.json:135]:
```suggestion
					"text": "Worker management"
```

**Review by yhyakuna — COMMENTED** [review]
Updated for Vault & Boundary

**Review by yhyakuna — APPROVED** [review]

**Pull request merged**